day3/part1.py

```diff
@@ -0,0 +1,37 @@
+if __name__ == "__main__":

```

Comment:
Holy shit.  You can start discussions on this thing

Reply:
Yeah it's really good for having debates or at least a back and forth on some decisions we had to make as programmers